docs/translations/api-docs/data-grid/grid-aggregation-function-data-source.json

```diff
@@ -0,0 +1,17 @@
+{
+  "interfaceDescription": "Grid aggregation function data source definition interface.",
+  "propertiesDescriptions": {
+    "columnTypes": {
+      "description": "Column types supported by this aggregation function.<br />If not defined, all types are supported (in most cases this property should be defined)."
+    },
+    "hasCellUnit": {
+      "description": "Indicates if the aggregated value have the same unit as the cells used to generate it.<br />It can be used to apply a custom cell renderer only if the aggregated value has the same unit."
```

Comment:
```suggestion
      "description": "Indicates if the aggregated value has the same unit as the cells used to generate it.<br />It can be used to apply a custom cell renderer only if the aggregated value has the same unit."
```